cli/src/pcluster/constants.py

```diff
@@ -254,6 +254,7 @@ class Feature(Enum):
     FSX_OPENZFS = "FSx OpenZfs"
     SLURM_DATABASE = "SLURM Database"
     CLUSTER_HEALTH_METRICS = "Cluster Health Metrics"
+    CAPACITY_BLOCK = "CAPACITY_BLOCK"
```

Comment:
This string is customer facing, so I suggest to use "Capacity Block" rather than "CAPACITY_BLOCK"